docs/weblog/README.md

```diff
@@ -875,7 +875,7 @@ c377db41-b664-4e30-af57-5df2e803bec7
 Examples:
 - `GET`: `/session/new`
 
-### \[GET\] /session/user
+### **UNUSED** \[GET\] /session/user
```

Comment:
Maybe we can just remove it as a separate PR

Reply:
I think we should remove it once all other libs have removed it :+1:

Reply:
otherwise imagine a dev debugging something and not finding docs for the endpoint he's working on, it's weird